src/raspberryEsp32Interface.cpp

```diff
@@ -155,7 +155,7 @@ namespace bzzz
 
     bool RaspberryEsp32Interface::armed()
     {
-        return m_encodedSwitchesData & RADIO_SWITCH_B_BIT;
+        return m_encodedSwitchesData == RADIO_SWITCH_B_BIT && throttleReferencePercentage() < 0.05 ;
```

Comment:
Is 5% low enough? Should we make it 0 instead?

Reply:
Yea 5% seems to be low enough (the motors don't spin), 

I didn't want to set it to zero encase the actual value sent from receiver isn't exactly 0% e.g. expects value 300 receives 301. anyway we should make this a variable as well in the config file